src/frameworks/fastify.js

```diff
@@ -19,7 +29,7 @@ function getValidator(validateRequest) {
 
     function _getParameters(req) {
         const requestOptions = {};
-        const path = req.raw.url;
+        const path = req.urlData().path;
```

Comment:
if we're going for the peer dependency solution, it's better to check that urlData is a function before calling `req.urlData().path` and fallback to `req.raw.url`